src/utilities/ses-transport.util.ts

```diff
@@ -1,5 +1,5 @@
 const nodemailer = require('nodemailer');
-const aws = require('@aws-sdk/client-ses');
+import aws = require('@aws-sdk/client-ses');
```

Comment:
same as above

Reply:
I am also getting the error "Module... has no default export." with this syntax:
`import aws from '@aws-sdk/client-ses';`

I also tried this, with the same error:
`import aws from '../../node_modules@aws-sdk/client-ses';`

Reply:
Solution was to do `import * as aws from...`